### PR TITLE
NFS-Ganesha: Add "Protocols" and "Transport" in ganesha.conf

### PR DESCRIPTION
**Pull request by supriti**
Signed-off-by: Supriti Singh <supriti.singh@suse.com>

### CODE REVIEW

File: srv/salt/ceph/ganesha/files/ganesha.conf.j2 [hunk 1]
```diff
@@ -8,6 +8,10 @@ EXPORT
 	Pseudo = /cephfs/;
 
 	Access_Type = RW;
+        
+        Protocols = 4;
```
Comment (rjfd) [inline srv/salt/ceph/ganesha/files/ganesha.conf.j2]: Shouldn't we specify the default, which is `3, 4`?

File: srv/salt/ceph/ganesha/files/ganesha.conf.j2 [hunk 2]
```diff
@@ -8,6 +8,10 @@ EXPORT
...
+
+        Transports = TCP;
```
Comment (rjfd) [inline srv/salt/ceph/ganesha/files/ganesha.conf.j2]: The same here
Shouldn't we specify the default, which is `TCP, UDP`?

**Comment by smithfarm** [comment]
@supriti Good idea to force NFSv4 by default.

nit: Could the white space be lined up? Maybe the original is using a tab character to indent, and your patch is using 8 spaces?

File: srv/salt/ceph/ganesha/files/ganesha.conf.j2 (hunk 2 above)
Comment (smithfarm) [inline srv/salt/ceph/ganesha/files/ganesha.conf.j2]: Is there any scenario in which NFSv4 would use UDP? (Not sure if it has a "compatibility" mode for communicating with NFSv2/3 UDP clients.)

Comment (smithfarm) [inline srv/salt/ceph/ganesha/files/ganesha.conf.j2]: in any case, `TCP, UDP` is fine with me

File: srv/salt/ceph/ganesha/files/ganesha.conf.j2 (hunk 1 above)
Comment (smithfarm) [inline srv/salt/ceph/ganesha/files/ganesha.conf.j2]: fine with me

Comment (supriti) [inline srv/salt/ceph/ganesha/files/ganesha.conf.j2]: Agreed. Updated.

**Pull request merged**